rllib/core/learner/learner.py

```diff
@@ -1189,6 +1368,23 @@ def _make_module(self) -> MultiAgentRLModule:
         module = module.as_multi_agent()
         return module
 
+    def _check_registered_optimizer(
```

Comment:
I couldn't find where we call this function.

Reply:
Totally, I erased the main call in `Learner.register_optimizer()` :) Fixed.